src/NuGet.Clients/NuGet.CommandLine/Commands/VerifyCommand.cs

```diff
@@ -21,6 +22,9 @@ protected VerifyCommand() : base()
             CertificateFingerprint = new List<string>();
         }
 
+        [Option(typeof(NuGetCommand), "VerifyCommandFingerprintAlgorithmDescription")]
```

Comment:
Validate that users do not pass an algorithm without a fingerprint.

Reply:
An algorithm could be passed without a fingerprint. This options are independent from each other in the sense that, if an algorithm is passed, the algorithm will be used to display a fingerprint computed with that algorithm whenever certificate information is displayed. If the list is passed it will be assumed that the algorithm used is the one given in that option.